src/docs/designs/planner.md

```diff
@@ -37,5 +37,15 @@ relative_path: ../../../
 | [Work Item - Hierarchy](https://redhat.invisionapp.com/share/ZPB1VOOXS)                             | Visual                | Sunil         | Ready for Dev     |             | [UX GitHub Issue 134](https://github.com/fabric8-ui/fabric8-ux/issues/134)
 | [Work Item - Detail Panel Holistic Review](https://redhat.invisionapp.com/share/4WB800RHD)          | Visual                | Brian         | In Progress       |             | [UX GitHub Issue 286](https://github.com/fabric8-ui/fabric8-ux/issues/286)
 | [Work Item - Modular Behavior](https://redhat.invisionapp.com/share/8QACKL3AY)                      | Wireframe             | Monica        | Ready for Dev     | 02/06/2017  | [UX GitHub Issue 314](https://github.com/fabric8-ui/fabric8-ux/issues/314)
+| [Hypothesis Engine](https://drive.google.com/open?id=1jYGo2zXqSCiyCsBCDtU2oQlmLR4OodfY)             | IA + IX               | Samir    | In Progress       | 10/01/2018  |
+| [Analytics-Stack Report UI Revamp](https://redhat.invisionapp.com/share/6TF4C40KF#/screens/270948052_Screen00)             | Wireframe + Visual Design               | Parvathy, Ashish    | Ready for Dev       | 10/01/2018  |
+| [Analytics-Stack Report UI Revamp (VD to Patternfly)](https://redhat.invisionapp.com/share/JTF8802X6)             | Visual Design               | Ashish    | Ready for Dev       | 10/01/2018  |
```

Comment:
What is the difference between this item and the one above? Which one is going to be used for development (as it is marked Ready for Dev). That should be the design that is shared, in order to avoid confusion.

Reply:
I have to check with Team

Reply:
Hi,
**[Analytics-Stack Report UI Revamp (VD to Patternfly)](https://redhat.invisionapp.com/share/JTF8802X6)** is the one which will be developed ,because the mockup in the other invision is not following Patterenfly styles.
The other one was made first and showcased to the stakeholders to get sign off,but the VD was not following patternfly.